packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -395,15 +438,20 @@ class _SnackBarState extends State<SnackBar> {
             ButtonTheme(
```

Comment:
ButtonTheme. UGH. This part of the implementation needs another look; future PR.

Reply:
yeah, snackbar is another one of those widgets to update with new button universe